--- src/main/java/com/godiddy/api/client/openapi/api/UniversalResolverApi.java   2025-08-09 15:23:20.886433176 +0200
+++ src/main/java/com/godiddy/api/client/openapi/api/UniversalResolverApi.java.changed   2025-08-09 15:23:15.398449795 +0200
@@ -165,7 +165,7 @@
     StringJoiner localVarQueryStringJoiner = new StringJoiner("&");
     String localVarQueryParameterBaseName;
     localVarQueryParameterBaseName = "options";
-    localVarQueryStringJoiner.add(options.toUrlQueryString());
+    if (options != null) localVarQueryStringJoiner.add(options.toUrlQueryString());

     if (!localVarQueryParams.isEmpty() || localVarQueryStringJoiner.length() != 0) {
       StringJoiner queryJoiner = new StringJoiner("&");